scripts/build.py

```diff
@@ -128,23 +128,26 @@
 
 HASH_BLOCK_SIZE = 2**20
 
+APP_DEV_YAML_FILE = 'app_dev.yaml'
```

Comment:
APP_DEV_YAML_FILEPATH, APP_YAML_FILEPATH

(There is a difference between the name/label/path of a file, and the file itself.)

Reply:
Done